File: Modul - 6/Soal - 2/PRAK602-2410817220009-NABILLAPUTRINUGRAHA.c
#include <stdio.h>
int main() {
    int ruangan;

    //buat nginput jumlah ruangan
    scanf("%d", &ruangan);

    int zetsu_counts[ruangan];
    int result[ruangan];

    //buat nginput jumlah zetsu putih di setiap ruangan
    for(int i = 0; i < ruangan; i++) {
        scanf("%d", &zetsu_counts[i]);
    }

    //buat ngitung jumlah zetsu putih setalah belah diri
    for(int i = 0; i < ruangan; i++) {
        result[i] = zetsu_counts[i] * (i + 1);
    }

    //buat nyetak hasil
    printf("\n");
    for(int i = 0; i < ruangan; i++) {
        printf("%d ", result[i]);
    }
    printf("\n");
    return 0;
}

File: Modul - 6/Soal - 2/PRAK602-2410817220009-NABILLAPUTRINUGRAHA.py
#fungsi buat ngitung jumlah zetsu putih
def hitung_zetsu(ruangan, zetsu_counts) :
    result = []
    for i in range(ruangan) :
        result.append(zetsu_counts[i] * (i + 1))
    return result

#buat nginput jumlah ruangan
ruangan = int(input(" "))

#buat nginput julah zetsu putih di setiap ruangan
zetsu_counts = list(map(int, input(" ").split()))

#buat ngitung jumlah zetsu putih setalah membelah diri
result = hitung_zetsu(ruangan, zetsu_counts)

print()
#buat nyetak hasil
print(' '.join(map(str, result)))